tests/phpunit/integration/MediaWiki/ExternalLinksTest.php

```diff
@@ -95,4 +96,20 @@ public function testPageIdsContainingNothing() {
 		$this->assertSame( [], $result );
 	}
 
+	public function testGetSingleIndex() {
+		$url = 'whatever';
+		$rows = $this->getExternalLinksRows( $url );
```

Comment:
I think this and the next line can be removed. It doesn’t matter what’s in the database – `LinkFilter::makeindexes()` isn’t supposed to access the database – the important thing is that `$url` is protocol-relative (which is why `makeIndexes()` returns two elements: one for HTTP, one for HTTPS).

Reply:
Done!.